src/transformers/utils/fx.py

```diff
@@ -582,8 +614,31 @@ def __contains__(self, key):
         return super().__contains__(key)
 
 
+class HFModelAttribute(HFProxy):
+    "Non tensor/module/parameter attributes"
+
+    def __getattr__(self, item) -> "HFModelAttribute":
+        # If `item` is a typical HFProxy attribute, `node` and `tracer` have to be manually added as they are only here in HFProxy instances
+        if item in set(HFProxy.__dict__) | {"node", "tracer"}:
+            return super().__getattribute__(item)
+
+        # Otherwise we attempt to obtain `_metadata` is available and get `item` there
+        try:
+            _metadata = self._metadata
+        except AttributeError:
+            _metadata = None
+        if _metadata is not None:
+            return getattr(_metadata, item)
+
+        # Without `_metadata` we create a proxy to get such an element.
+        return self.tracer.create_proxy(
+            "get_attr", item, (), {}, proxy_factory_fn=lambda node: HFModelAttribute(node, self.tracer)
+        )
+
+
 class HFAttribute(HFProxy):
     def __init__(self, root, attr: str):
+        assert attr != "_metadata", "`_metadata` is an invalid attribute for a proxy"
```

Comment:
No assert in `transformers`
```suggestion
         if attr == "_metadata":
             raise AttributeError("_metadata is an invalid attribute for a proxy")
```